### PR TITLE
Feature eng 1338

### PR DESCRIPTION
**Pull request by jabelardo**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2,10 +2,11 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.optimaize.languagedetector</groupId>
+    <!-- FORKED FROM <groupId>com.optimaize.languagedetector</groupId> -->
+    <groupId>com.zignallabs</groupId>
     <artifactId>language-detector</artifactId>
     <name>language-detector</name>
-    <version>0.6-SNAPSHOT</version>
+    <version>0.6.1-SNAPSHOT</version>
```
Comment (jakehschwartz) [inline pom.xml]: This should probably bump to 0.7.0-SNAPSHOT

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  java:
+    version: oraclejdk8
+
+deployment:
+  develop:
+    branch: develop
+    commands:
+      -  mvn deploy
+  release:
+    tag: /v[0-9]+(\.[0-9]+){0,2}(-rc)?/ # v1.2.3 or v1.2.3-rc
```
Comment (jakehschwartz) [inline circle.yml]: I thought we decided we wanted to do branch based deploys, not tag/RC based deploys?

Comment (jabelardo) [inline circle.yml]: I understood it was branch based for develop and tag based for qa and master. But now I realize you are right. I'm fixing this

**Comment by jakehschwartz** [comment]
+1

**Comment by jamestyack** [comment]
Looks good, thanks Jose

**Pull request merged**